playerbot/WorldPosition.cpp

```diff
@@ -610,7 +610,7 @@ bool WorldPosition::loadMapAndVMap(uint32 mapId, uint32 instanceId, int x, int y
         return false;
 
 #ifdef MANGOSBOT_ZERO
-    MMAP::MMapFactory::createOrGetMMapManager()->loadMap(mapId, x, y);
+    MMAP::MMapFactory::createOrGetMMapManager()->loadMap(sWorld.GetDataPath(), mapId, x, y);
 #endif
 #ifdef MANGOSBOT_ONE
```

Comment:
I kept these separate since the MANGOSBOT_ONE does extra things the MANGOSBOT_ZERO does not but can they be unified now that the loadMap has been synced?

Reply:
It'll probably need the same logic as MANGOSBOT_ONE since the loadMap as an assert for instances. I'll run a test to see how it pans out.